src/constants/prop-types.js

```diff
@@ -47,7 +47,14 @@ const elementData = PropTypes.shape({
   ]).isRequired,
 })
 
+const image = PropTypes.shape({
+  url: PropTypes.string.isRequired,
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+})
+
 export default {
   elementColors,
   elementData,
+  image,
```

Comment:
Since we separate components into different folders, such as `body`, `image`, `aside`, etc., could we move these constants into folders?
`image` -> `src/components/image/prop-types.js`
`elementData` -> `src/components/body/prop-types.js` 

Only leave the global settings such as `elementColors` here?